lib/matplotlib/lines.py

```diff
@@ -1432,7 +1432,8 @@ def __init__(self, xy1, xy2, slope, **kwargs):
 
     def get_transform(self):
         ax = self.axes
-        points_transform = self._transform + ax.transData.inverted()
+        points_transform = self._transform + ax.transData.inverted()\
+                           + ax.transScale
```

Comment:
I think we typically do this:
```suggestion
        points_transform = (self._transform + ax.transData.inverted() + 
                                          ax.transScale)
```

Reply:
^Sorry, align "ax.transScale" under "self._transform"

Reply:
thanks, fixed in 025b2ee